packages/forms/resources/views/components/builder.blade.php

```diff
@@ -133,8 +138,6 @@
                                         'dark:text-gray-400' => config('forms.dark_mode'),
                                     ])>
                                         @php
-                                            $block = $item->getParentComponent();
-
                                             $block->labelState($item->getRawState());
```

Comment:
Wait... I think this feature already exists using the existing `label()` method on the block.

Reply:
Lol. You are totally right! I was so focused on the Builder level, that I never thought about the block level, because this is how it works on the Repeater. 

So my complete PR is useless. Only missing feature is the HtmlString instead of just plain string. Can/should I close this PR and create a new one to add HtmlString to both - Block label and Repeater itemLabel?

Reply:
Sure!

Reply:
Or you can use this PR, it doesn't matter